lego/api/v1.py

```diff
@@ -210,4 +215,5 @@
 urlpatterns = [
     path("", include(router.urls)),
     path("forums/", include((forums_urls, "forums"))),
+    path("", include((lending_urls, "lending"))),
```

Comment:
Why do we add this here instead of just registering it with the router above?

Reply:
I think it's because the routing structure is a little different than what the viewset would indicate